modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/layout-page-templates/display-page-templates/app-detail/page-definition.json

```
@@ -208,14 +207,17 @@
 													{
 														"definition": {
 															"fragment": {
-																"key": "BASIC_COMPONENT-heading"
+																"key": "text",
+																"siteKey": "Marketplace"
```

Please make the replacement
```suggestion
																"siteKey": "[$GROUP_KEY$]"
```